.github/workflows/ci.yaml

```diff
@@ -18,7 +17,15 @@ jobs:
        python-poetry: 'false'
     secrets: inherit
 
-  ci:
+  UnitTest:
+    name: Python Unit Test with Postgres
+    uses: uc-cdis/.github/.github/workflows/python_unit_test.yaml@master
+    with:
+       setup-script: 'tests/ci_setup.sh'
+       test-script: 'tests/ci_commands_script.sh'
```

Comment:
since these 2 values are the same as the defaults, you could remove them from here, unless you want it to be more explicit where the files are used

Reply:
Yeah I figured since: a) I was gonna use this as an example PR for others to base theirs and b) it's clearer how it's working this way, it'd be better to be explicit